src/nxdoc/nuxeo-server/rest-api/rest-api-web-adapters.md

```diff
@@ -232,6 +232,29 @@ Default adapters provided by default.
   </tr>
 </table>
 
+<table class="hover">
+  <tr>
+    <td class="small-2">**@emptyWithDefault**</td>
+    <td></td>
+  </tr>
+  <tr>
+    <td>
+      Returns an empty document initialized with default properties. <br />
+      The target document acts as the parent of the empty document. <br />
+      Query parameters: <br />
+      `type`: required, type of the empty document.<br />
+      `name`: optional, name of the empty document. <br />
+      <br />
+      Note that calling `emptyWithDefault` adapter triggers the `emptyDocumentModelCreated` core event, so the returned empty document will have all proeprties set by listeners listening `emptyDocumentModelCreated` as well.
```

Comment:
- proeprties -> properties
- listening `emptyDocumentModelCreated` -> listening **to** `emptyDocumentModelCreated`